unfoldingWord/translationCore
# Feature - #2130 - Auto-focuses on the Username field of Door43 and Guest Login

#### This pull request addresses:

Addresses the issue #2130 


#### How to test this pull request:

Logout (if you are already logged in) and then go to the login selection screen. Click on "Door43" and it should have the Username field selected. Go back and click to login as a guest. It should also have the Username field focused.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3208)
